blocks/components/editable/index.js

```diff
@@ -266,10 +268,13 @@ export default class Editable extends wp.element.Component {
 			this.focus();
 		}
 
+		// The savedContent var allows us to avoid updating the content right after an onChange call
```

Comment:
if you make a change on the editable. Say the old value is `a` and the new one is `ab`, we don't want to trigger the `updateContent` even if the old value prop and the new one are different. That's what I'm trying to achieve here by caching the  `savedContent`.